public/locales/en/company.json

```
@@ -0,0 +1,88 @@
+{
+  "head": {
+    "title": "Company Overview | Anti-Pattern Inc.",
+    "description": "This page is the corporate website of Anti-Pattern Inc. We introduce our corporate philosophy, services, company overview, and more."
+  },
+  "outline": {
+    "outline1": [
+      {
+        "title": "Company Name",
+        "content": "Anti-Pattern Inc."
+      },
+      {
+        "title": "Date of Foundation",
+        "content": "July 1, 2019"
+      },
+      {
+        "title": "Representative",
+        "content": "Yuki Ozasa"
+      },
+      {
+        "title": "Location",
+        "content": "3-15-9 Minowa Omotesando 3F, Minato-ku, Tokyo"
+      }
+    ],
+    "outline2": [
+      {
+        "title": "Phone Number",
+        "content": "+81-3-6885-6136"
+      },
+      {
+        "title": "Number of Employees",
+        "content": "46 (as of July 1, 2024)"
+      },
+      {
+        "title": "Capital",
+        "content": "50 million yen"
+      },
+      {
+        "title": "Major Shareholder",
+        "content": "Yuki Ozasa"
+      }
+    ]
+  },
+  "members": [
+    {
+      "img": "Ozasa",
```

読み込み先で画像をimportするより、ここにパスを明記したほうが読みやすい気がする
```suggestion
      "img": "'images/ozasa.jpg'",
```

---

nextjsの仕様の問題で、画像がpublicディレクトリにないため、画像を直接呼び出すことはできないようです。
なので今回はStaticImageDataを使用しました。
StaticImageDataを利用すると画像の自動最適化やレスポンシブ画像が適用されてパフォーマンスが向上するので今のままの方が良いのではないかと思うのですがどうでしょうか？

---

publicにないのか！失礼！